src/routes/api/weather/index.ts

```diff
@@ -84,11 +83,13 @@ export const GET: RequestHandler = async (requestEvent: RequestEvent) => {
     .then((r: Response) => r.json())
     .then((r: JSONObject) => ({
       status: 200,
-      body: plainToInstance(ResponseDTO, { units: query.units, ...r }, classTransformOptions),
+      body: plainToInstance(ResponseDTO, { units: query.units, ...r }),
       headers: {
         'Cache-Control': 'public, max-age=600',
         'Expires': new Date(Date.now() + 10 * 60 * 1000).toUTCString()
       }
     }))
-    .catch((e) => onFailure(e));
+    .catch((e) => {
+      console.log(e);
+    });
```

Comment:
This breaks the async/await block in the weather component. User will no longer see error with proper message when the API fails.